airflow/contrib/operators/gcp_bigtable_operator.py

```diff
@@ -0,0 +1,376 @@
+    REQUIRED_ATTRIBUTES = ('project_id', 'instance_id')
+    template_fields = ['project_id', 'instance_id']
+
+    @apply_defaults
+    def __init__(self,
+                 project_id,
+                 instance_id,
+                 *args, **kwargs):
+        self.project_id = project_id
+        self.instance_id = instance_id
+        self._validate_inputs()
+        self.hook = BigTableHook()
+        super(BigTableInstanceDeleteOperator, self).__init__(*args, **kwargs)
+
+    def execute(self, context):
+        try:
+            self.hook.delete_instance(self.project_id, self.instance_id)
+        except google.api_core.exceptions.NotFound:
+            self.log.info("The Instance '{}' does not exist in project '{}'.".format(self.instance_id, self.project_id))
+        except google.api_core.exceptions.GoogleAPICallError as e:
+            self.log.error('An error occurred. Exiting.')
+            raise e
+
+
+class BigTableTableCreateOperator(BaseOperator, BigTableValidationMixin):
+    """
+    Creates the table in BigTable instance.
+
+    For more details about creating table have a look at the reference:
+    https://googleapis.github.io/google-cloud-python/latest/bigtable/table.html#google.cloud.bigtable.table.Table.create
+
+    :type project_id: str
+    :param project_id: The Google Developers Console [project ID or project number]
+    :type instance_id: str
+    :param instance_id: The ID of the instance that will hold the new Table
+    :type table_id: str
+    :param table_id: The ID of the Table to be created
+    """
+    REQUIRED_ATTRIBUTES = ('project_id', 'instance_id', 'table_id')
+    template_fields = ['project_id', 'instance_id', 'table_id']
+
+    @apply_defaults
+    def __init__(self,
+                 project_id,
+                 instance_id,
+                 table_id,
+                 initial_split_keys=None,
+                 column_families=None,
+                 *args, **kwargs):
+        self.project_id = project_id
+        self.instance_id = instance_id
+        self.table_id = table_id
+        self.initial_split_keys = initial_split_keys or list()
+        self.column_families = column_families or dict()
+        self._validate_inputs()
+        self.hook = BigTableHook()
+        self.instance = None
+        super(BigTableTableCreateOperator, self).__init__(*args, **kwargs)
+
+    def _compare_column_families(self):
+        table_column_families = self.hook.get_column_families_for_table(self.instance, self.table_id)
+        if table_column_families.keys() != self.column_families.keys():
+            self.log.error("Table '{}' has different set of Column Families".format(self.table_id))
+            self.log.error("Expected: {}".format(self.column_families.keys()))
+            self.log.error("Actual: {}".format(table_column_families.keys()))
+            return False
+
+        for key in table_column_families.keys():
+            if table_column_families[key].gc_rule != self.column_families[key]:
```

Comment:
Why do we compare_gc_rule with family? It does not seem right (or needs explanation if it makes sense)

Reply:
I added appropriate explanation

Reply:
I think we can explain more about column_families parameter in the init method. Same as the link you put for [Table.Create method ](https://googleapis.github.io/google-cloud-python/latest/bigtable/table.html#google.cloud.bigtable.table.Table.create)

"column_failies – (Optional) A map columns to create. The key is the column_id str and the value is a GarbageCollectionRule"

Reply:
probably need comment for `initial_split_keys` too.